bayesian/api_v1.py

```diff
@@ -71,6 +72,11 @@
 TOTAL_COUNT_KEY = 'total_count'
 
 ANALYTICS_API_VERSION = "v1.0"
+HOSTNAME = os.environ.get('HOSTNAME', 'bayesian-api')
+METRICS_SERVICE_URL = "http://{}:{}".format(
+    os.environ.get('METRICS_COLLECTOR_HOST', 'metrics-accumulator'),
```

Comment:
@miteshvp  The environment variables have to be set as `METRICS_ACCUMULATOR_HOST/_PORT` If the service name is `metrics-accumulator`.